src/components/molecules/Settings/Workspace/Plugin/PluginSection/index.tsx

```diff
@@ -1,15 +1,19 @@
-import React from "react";
+import React, { useState } from "react";
 
-import { styled, useTheme } from "@reearth/theme";
+// import { styled, useTheme } from "@reearth/theme";
```

Comment:
These commented parts will be uncommented unless the full plugin feature is released.